Routines/_ut.m

```diff
@@ -494,3 +499,11 @@ D GETLIST(.%utROU,1,%ut("RSLT"))
  ;
  N %ZH2 S %ZH2=T1-T0*1000
  QUIT %ZH2
+ ;
+WTRACE ; Write a stack trace.
+ Q:$$GETSYS()'=47 ; Code only tested on GTM
+ Q:'%utVERB ; Disable if not verbose
```

Comment:
Can you increment the version at the top of the file to 1.63?

Reply:
Done.